airbyte-integrations/connectors/source-commercetools/source_commercetools/schemas/shared/taxed_item_price.json

```diff
@@ -0,0 +1,11 @@
+{
+  "type": ["null","object"],
+  "properties": {
+    "totalNet": {
+      "$ref": "money.json"
+    },
+    "totalGross": {
+      "$ref": "money.json"
+    }
+  }
+}
```

Comment:
@jacqueskpoty you need to run `./gradlew format` to apply formatting. This is going to break master building. Could you provide this?

Reply:
@marcosmarxm This is normally part of commercetools source. Another source I contributed but has its own PR. So i will remove it from this PR so we can only deal with WooCommerce here